src/java/io/bazel/rulesscala/scalac/ScalaCInvoker.java

```diff
@@ -255,6 +255,37 @@ private static void processRequest(List<String> args) throws Exception {
           reporter.flush();
           throw new RuntimeException("Build failed");
       } else {
+        /**
+         * See if there are java sources to compile
+         */
+        if (ops.javaFiles.length > 0) {
+          StringBuilder cmd = new StringBuilder();
+          cmd.append(ops.javacPath);
+          if (ops.jvmFlags != "") cmd.append(ops.jvmFlags);
```

Comment:
(Miss filter/flatMap don't you!)

ok this seems fine since it must work. (In some other of these sub process libs they can require that the args are passed to it and the 'process' is a executable to call than a string with args in it... indeed i believe scala's works like this.)